gcs/upload.py

```diff
@@ -241,9 +241,13 @@ def init_write_object_grpc(cls, db, request_iterator, context):
             upload.metadata.metadata["x_emulator_no_md5"] = "true"
         return upload, is_resumable
 
-    def resumable_status_rest(self):
+    def resumable_status_rest(self, no_308=False):
```

Comment:
`no_Foo = False` is a code smell. Why two negations to express what we want? Consider `use_308=True`

And does this have to have a default value?  It seems that we would want to change all call sites to pass the right parameter in?

Reply:
Well noted! With that in mind, I'd like to use `override_308=False`

Considering how GCS works or at least how it's documented publicly, I think it makes sense to have a default value `override_308=False`. IIUC, `308 Resume Incomplete` is the default GCS response; `308` is overriden by `200` under resume incomplete only when the request header `X-Guploader-No-308=yes` is included. Does that seem reasonable?

Reply:
Updated with proposed modifications for now. Happy to make further changes!

Reply:
Still weird that the default is to say "please don't do X" (with `X==override_308`), but I think your argument about consistency with the GCS headers is compelling.  LGTM